src/js/utils/StringUtil.js

```diff
@@ -215,6 +215,16 @@ const StringUtil = {
     }, []);
 
     return this.idToTitle(splitID, splitBy, replace, removeConsecutive);
+  },
+
+  lrtrim(str, trimStr = " ") {
+    if (typeof str === "string") {
+      return str
```

Comment:
Some comments -
1. Think we should use `+` instead of `*` since `+` will only match if there is at least 1 char. `*` will always match.
2. We could combine into a single RegExp: 
`.replace(new RegExp("^[" + trimStr +" ]+|[" + trimStr + "]+$", "g"), "");`